modules/serverless/pages/features/transport-encryption/transport-encryption-setup.adoc

```diff
@@ -0,0 +1,226 @@
+= Setup transport encryption in Eventing
+:compat-mode!:
+// Metadata:
+:description: Setup {serverlessproductname} transport encryption for Eventing
+
+This page describes the transport encryption feature which allows transporting data and events over secured and encrypted HTTPS connections using TLS.
+
+[IMPORTANT]
+====
+{serverlessproductname} transport encryption for Eventing is a Developer Preview feature only.
+Developer Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/.
+====
+
+== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the {serverlessoperatorname}.
+
+* Install the {certmanageroperatorname}.
+
+== Setup a `SelfSigned` `ClusterIssuer`
+
+> `ClusterIssuers`, are Kubernetes resources that represent certificate authorities (CAs) that are able to generate signed certificates by honoring certificate signing requests.
```

Comment:
This seems rendered indented. Should this be a info block or something?

Reply:
It is supposed to be a citation but info block sounds good